### PR TITLE
D8 core 971 gutter

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW

# Summary
Adjust the space between the aside and the main content on Stanford pages 

# Needed By (Date)
- RC1

# Urgency
- Nice to have?

# Steps to Test
- Checkout this branch
- Verify that for breakpoints xl and larger, there's an extra 150px between the main content and the side bar.

## Smaller than xl
Screen Shot 2019-12-20 at 7 58 25 PM

## XL
Screen Shot 2019-12-20 at 7 58 58 PM

## XXL
Screen Shot 2019-12-20 at 7 59 19 PM

# Affected Projects or Products
D8Core

# Associated Issues and/or People
- JIRA ticket
https://stanfordits.atlassian.net/browse/D8CORE-971

- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -63,7 +63,8 @@ var webpackConfig = {
     "lockup":               path.resolve(__dirname, srcSass + "/components/lockup.component.scss"),
     "logo":                 path.resolve(__dirname, srcSass + "/components/logo.component.scss"),
     "media":                path.resolve(__dirname, srcSass + "/components/media.component.scss"),
-    "quote":                path.resolve(__dirname, srcSass + "/components/quote.component.scss")
+    "quote":                path.resolve(__dirname, srcSass + "/components/quote.component.scss"),
+    "layouts":              path.resolve(__dirname, srcSass + "/layouts/index.scss")
```
Comment (cjwest) [inline webpack.config.js]: Hi @sherakama,

This is a start to get things working.

Two questions for you:
1) Sanity check: Does my approach make sense? Does putting this work in the layout make sense? 

2) I'm missing a step to get this to the browser. Can you see what it is I am missing?
It creates a file in /dist, but that file doesn't show up on the page in the browser.

File: lib/scss/jumpstart_ui.layout.scss [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
-//
-// // Render Decanter Library Base Styles Only.
-// @import 'decanter/core/src/scss/layout/index';
-//
-// // Jumpstart UI Custom Styles
-// @import './layouts/index';
+@import '~decanter/core/src/scss/decanter-no-markup';
+
+// Render Decanter Library Base Styles Only.
+@import '~decanter/core/src/scss/layout/index';
```
Comment (sherakama) [inline lib/scss/jumpstart_ui.layout.scss]: Do you need these?

File: lib/scss/layouts/stanford-page.layout.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@charset 'UTF-8';
+@import '../jumpstart_ui.config';
+
+.node-stanford-page-su-page-components {
+  @media #{$su-breakpoint-xl} {
```
Comment (sherakama) [inline lib/scss/layouts/stanford-page.layout.scss]: This should use the `@import grid-media('xl') {` mixin format.

File: lib/scss/layouts/stanford-page.layout.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+    padding-left: 150px;
```
Comment (sherakama) [inline lib/scss/layouts/stanford-page.layout.scss]: This should use the `@import padding()` mixin and the `$su-screen-margins` variable map.

**Comment by sherakama** [comment]
@cjwest thanks for this work. Feedback has been posted and revisions requested.

**Comment by sherakama** [comment]
@cjwest ping.

File: lib/scss/jumpstart_ui.layout.scss (hunk 2 above)
Comment (cjwest) [reply lib/scss/jumpstart_ui.layout.scss]: Without `@import '~decanter/core/src/scss/decanter-no-markup';` I get an error (see below).
However, it appears that I don't need `@import '~decanter/core/src/scss/layout/index';`

Would it make better sense to move that into 'jumpstart_ui.config' ?

```
ERROR in ./lib/scss/jumpstart_ui.layout.scss
    Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
    ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
    SassError: Undefined variable.
      ╷
    5 │   @media #{$su-breakpoint-xl} {
      │            ^^^^^^^^^^^^^^^^^
      ╵

```

File: lib/scss/layouts/stanford-page.layout.scss (hunk 4 above)
Comment (cjwest) [reply lib/scss/layouts/stanford-page.layout.scss]: The $su-screen-margins doesn't have a size that is equivalent to 150px.
https://decanter.stanford.edu/docs/scss/index.html#variable-$su-screen-margins

File: lib/scss/layouts/stanford-page.layout.scss (hunk 3 above)
Comment (cjwest) [reply lib/scss/layouts/stanford-page.layout.scss]: Got it!

**Comment by cjwest** [comment]
@sherakama Thanks for your feedback. Please see my responses above.

**Comment by sherakama** [comment]
Ah, ok, this looks good now. Unfortunately, I don't think this work belongs here but rather in stanford_profile where we are already adjusting the looks and layout for that content type.

**Comment by sherakama** [comment]
Closing and moving to: https://github.com/SU-SWS/stanford_profile/pull/65

File: lib/scss/layouts/stanford-page.layout.scss (hunk 4 above)
Comment (sherakama) [reply lib/scss/layouts/stanford-page.layout.scss]: Ah, wrong gutter. My bad.

**Pull request closed without merging**